iroh/bench/src/lib.rs

```diff
@@ -70,9 +70,8 @@ pub struct Opt {
     pub initial_mtu: u16,
     /// Whether to run a local relay and have the server and clients connect to that.
     ///
-    /// Can be combined with the `DEV_RELAY_ONLY` environment variable (at compile time)
+    /// Can be combined with the `relay_only` configuration option on the endpoint
     /// to test throughput for relay-only traffic locally.
-    /// (e.g. `DEV_RELAY_ONLY=true cargo run --release -- iroh --with-relay`)
     #[clap(long, default_value_t = false)]
     pub with_relay: bool,
```

Comment:
Thanks for adjusting the docs here, but these docs are actually the CLI help text.
So saying "hey you can change the endpoint config" doesn't really help when you're reading this as CLI text :grimacing: 

I'd appreciate if there'd be two possible solutions:
1. Add another `pub only_relay: bool` option, so you can run `cargo run --release -- iroh --only-relay` (and `only-relay` should imply `with-relay`) or
2. Rename `with_relay` to `only_relay` and make that also set `.relay_only(true)` on the endpoint.

Reply:
Went with option 2 but am not happy about it as it depends on the test/test-utils feature to be active. I'll revert it to option 1.

Reply:
Nice!
Only other thing that might be nice is documenting that you need the `local-relay` feature to be active (and telling the user how to run this thing, so `cargo run --release --features=local-relay -- iroh --only-relay`).